index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NoteBook</title>
+    <style>
+        *{
+            margin: 0;
+            padding:0;
+            box-sizing: border-box;
+        }
+
+        body{
+            min-height: 100vh;
+            background:#d3d3d3;
+            font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+            display:flex;
+            justify-content:center;
+            align-items:center;
+        }
+        ::-webkit-scrollbar{
+            display: none;
+        }
+        .paper{
+            position:relative;
+            width: 90%;
+            max-width:800px;
+            min-width:400px;
+            height:480px;
+            margin: 0 auto;
+            background:#fafafa;
+            box-shadow:0px 2px 8px rgba(0,0,0,3);
+            border-radius:10px;
+            overflow: hidden;
+        }
+        .paper::before{
+            content:"";
+            position: absolute;
+            top:0;
+            bottom:0;
+            left:0;
+            width:55px;           
+            background:radial-gradient(#575450 6px,transparent 7px);
+            border-right:3px solid #d44147;
+            background-size: 40px 40px ;
+            box-sizing:border-box;
+        }
+        .paper-content{
+            position:absolute;
+            top:30px;
+            right:0;
+            bottom:30px;
+            left:60px;
+            background:linear-gradient(transparent, transparent 28px, #91d1d3 28px);
+            background-size : 30px 30px;
+        }
+        .paper-content textarea{
+            width:100%;
+            max-width:100%;
+            height: 100%;
+            max-height: 100%;
+            line-height: 30px;
+            background:transparent;
+            padding: 0 10px;
+            border:0;
+            outline:0;
+            color:mediumblue;
+            font-family:cursive;
+            font-weight: bold;
+            font-size:22px;
+            box-sizing: border-box;
```

Comment:
what happens if you remove this box-sizing property??

Reply:
I did'nt see any change can you explain?

Reply:
Exactly you didn't saw any change because you already use same property in universal selector at the top of css. Make sure to not repeat code in coding.

Reply:
Ok,Thanks